### PR TITLE
(Publish on 12-21-2023?) What's new: errors inbox mobile grouping

### PR DESCRIPTION
**Pull request by ken-barrette**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* Text for what's new information related to errors inbox mobile grouping
* Target publish date of 12-21-2023 in the morning
* This is a minor release created by the PMM so it does NOT need review by other PMMs

### CODE REVIEW

File: src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+releaseDate: '2023-12-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work'
+---
+
+New Relic errors inbox now lets you automatically group mobile error events into common patterns, reducing noise, allowing you to focus on identifying and resolving the issues with the highest user impact before they hurt the business. With New Relic errors inbox mobile grouping now you can:
+* **Identify high-impact errors faster:** Automatically identify patterns across thousands of individual errors and turn them into meaningful groups, so you can zero in on the most important issue and priorities. 
+* **Analyze more errors with less effort:** Improve user experience for apps with high-volume mobile engagement while staying under cardinality limits so you can capture and examine the errors that matter most
+* **Improve error triage and issue ownership:** Streamline root cause analysis with better insights into mobile user activity to expand full stack coverage with mobile error grouping is part of the New Relic all-in-one observability platform.
```
Comment (rhetoric101) [inline src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md:11]: @ken-barrette I'm not quite following this bullet point. Could we split it into two sentences with something like this:

```
* **Improve error triage and issue ownership:** Streamline root cause analysis with better insights into mobile user activity.  This expands full stack coverage with mobile error grouping as part of the New Relic all-in-one observability platform.
```

File: src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Streamline error management with New Relic errors inbox mobile grouping'
+summary: 'Automatically group thousands of error events into common patterns for faster triage of mobile user issues'
+releaseDate: '2023-12-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work'
+---
+
+New Relic errors inbox now lets you automatically group mobile error events into common patterns, reducing noise, allowing you to focus on identifying and resolving the issues with the highest user impact before they hurt the business. With New Relic errors inbox mobile grouping now you can:
+* **Identify high-impact errors faster:** Automatically identify patterns across thousands of individual errors and turn them into meaningful groups, so you can zero in on the most important issue and priorities. 
+* **Analyze more errors with less effort:** Improve user experience for apps with high-volume mobile engagement while staying under cardinality limits so you can capture and examine the errors that matter most
```
Comment (rhetoric101) [inline src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md:10]: @ken-barrette would you consider splitting up this sentence like this?

```
* **Analyze more errors with less effort:** Improve the user experience for apps with high-volume mobile engagement while staying under cardinality limits. This allows you to capture and examine the errors that matter most.
```

File: src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+* **Improve error triage and issue ownership:** Streamline root cause analysis with better insights into mobile user activity to expand full stack coverage with mobile error grouping is part of the New Relic all-in-one observability platform.
+
+To learn more about error gropuing check out our [documentation](https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work)
```
Comment (rhetoric101) [inline src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md:13]:
```suggestion
To learn more about error grouping, check out our [documentation](https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work).
```

**Review by rhetoric101 — COMMENTED** [review]
@ken-barrette I made a few suggestions you might consider.

File: src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md (hunk 2 above)
Comment (ken-barrette) [reply src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md:10]: Yes, I will make that change.

File: src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md (hunk 1 above)
Comment (ken-barrette) [reply src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md]: Done

**Review by rhetoric101 — APPROVED** [review]
@ken-barrette thanks for making these changes!

**Comment by ken-barrette** [comment]
This post does not need an additional PMM review since it was created by a PMM.

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**